### ISSUE TITLE
login with normal docker account is not working

### ISSUE THREAD

**Issue by giovixo97** [play-with-docker/play-with-docker#621]
Happy evening to everyone, i'm sorry to bother you but i'm facing some issue with login. If i try the Docker Hub credentials, in the developer console of any browser throw me this error:

`Failed` to load resource: the server responded with a status of 401 ()
(index):85 ERROR
Object
 config: headers: Accept: "application/json, text/plain, */*" [[Prototype]]: Object json Callback Param: "callback"method: "GET" paramSerializer: ƒ (a) transform Request: [ƒ] transform Response: [ƒ] url: "/users/me" [[Prototype]]: Object data: "" headers: ƒ (d) status: 401 statusText: "" xhrStatus: "complete" [[Prototype]]: Object`
 
 Can you try to take a look? If i use Google account i can login, only with docker account cannot login. Have a great day ^_^

**Comment by marcosnils** [comment]
hey there. Is this play-with-docker, or play-with-kubernetes?

**Comment by giovixo97** [comment]
play with docker, sorry for not specifying

**Comment by giovixo97** [comment]
![Cattura](https://github.com/play-with-docker/play-with-docker/assets/13524984/d2aef541-43bf-478f-873f-0a7ae93679c0)
this is the situation: if i choose Google, i can login succesfully, if i use docker it give me 401 error as the previous message

**Comment by marcosnils** [comment]
> this is the situation: if i choose Google, i can login succesfully, if i use docker it give me 401 error as the previous message

 I see, thanks. Strange since that works for me. By any chance do you have 2fa enabled for user/pass login? Can you check if enabling that helps?

**Comment by giovixo97** [comment]
i have 2FA but unfortunately it doesn't help

**Comment by giovixo97** [comment]
Happy morning, i'm here to update on the situation of play-docker. Now, i can't login with google and github. The server page return me error 500 every time.

**Comment by d3cryptofc** [comment]
@giovixo97 Despite the 500 Internal Server Error, the login is carried out, just go back to the home page and click on Play, but it would actually be good if the error was corrected.

**Comment by marcosnils** [comment]
> @giovixo97 Despite the 500 Internal Server Error, the login is carried out, just go back to the home page and click on Play, but it would actually be good if the error was corrected.

would you mind uploading a video about the behavior? I can't seem to be able to reproduce.

**Comment by giovixo97** [comment]
https://github.com/play-with-docker/play-with-docker/assets/13524984/a544f2f8-966a-4c40-92af-f8f8b21700c0

**Comment by d3cryptofc** [comment]
@marcosnils

In fact the error 500 Internal Server Error only occurs to me when I generate the OAuth URL manually and open it in a tab:

```bash
curl -sD - https://labs.play-with-docker.com/oauth/providers/docker/login | grep location
```

**Comment by giovixo97** [comment]
exaclty, if i visit the url manually i obtain error 500
![image](https://github.com/play-with-docker/play-with-docker/assets/13524984/98dd4252-0a6c-413a-ac14-7c1e6e448a54)

**Comment by d3cryptofc** [comment]
@giovixo97 Regarding the 401 error, could you tell me if docker labs has any cookies created in your browser?

Devtools > Application > Storage > Cookies > `https://labs.play-with-docker.com`

If there is a cookie called id, try to delete it with the right mouse button

**Comment by giovixo97** [comment]
but i'm using chrome incognito, cookie are always cleared (incognito means that i will never have cookie)

**Comment by d3cryptofc** [comment]
Sorry for my lack of attention, you're right, I used the cookie you sent and it really isn't recognized as authenticated, I'm really running out of ideas what could be happening to you 🤔

**Comment by giovixo97** [comment]
What i'm trying ti Say from the beginning, Is that someone Who have access to the backend service have to check the status of the server. This Is not my problem in my opinion, Is a global problem i think. I think that from my end Is impossible to solve. This Is only my opinion, if you know Who host this service please contact him and let him know about this situation

**Comment by d3cryptofc** [comment]
I agree with you regarding the 500 error, as it must have certainly raised an exception in the server logs, something that only they can resolve, but regarding the 401 error, not even I could replicate it on my machine, perhaps it is something specific

**Comment by giovixo97** [comment]
I have the same situation in Android, I've tried on multiple completely different devices and have the same problem. Even with different browsers I tried. in my opinion the 401 error is related to the 500 error. if I can't login then I have no authorization. Let me know of you want more feedback from my host

**Comment by d3cryptofc** [comment]
If the device is not the problem, then it is some particularity of your specific Docker account, because see, I created an account and recorded the procedure trying to replicate:

https://github.com/play-with-docker/play-with-docker/assets/47941854/cc468c9d-4320-4998-b982-f172937edf5a

**Comment by giovixo97** [comment]
I'm trying to investigate my account setting but i cannot find any particular problem
